app/services/atmosphere/cep/esper.rb

```diff
@@ -0,0 +1,18 @@
+module Atmosphere
+  class Cep::Esper
+    def add_event_type(ev_type_id, ev_definition)
+
+    end
+
+    def remove_event_type(ev_type_id)
+    end
+
+    def subscribe(complex_ev_definition)
```

Comment:
Unused method argument - `complex_ev_definition`. If it's necessary, use `_` or `_complex_ev_definition` as an argument name to indicate that it won't be used. You can also write as `subscribe(*)` if you want the method to accept any arguments but don't care about them.